bezier-rs/lib/src/lib.rs

```diff
@@ -125,37 +135,44 @@ impl Bezier {
 		};
 	}
 
+	/// Get the coordinates of the bezier segment's start point.
 	pub fn get_start(&self) -> DVec2 {
```

Comment:
A note on convention in the wider rust codebase: We avoid using the word `get` when possible with function naming. Instead we just remove the `get_` and have it be `start`, `end`, etc. We still use the word `set` when required for clarity. This is a light styling suggestion and taken with a grain of salt. <3